integration/src/test/java/info/rmapproject/indexing/kafka/IndexingConsumerIT.java

```diff
@@ -1,42 +1,32 @@
 package info.rmapproject.indexing.kafka;
 
 import static info.rmapproject.indexing.IndexUtils.EventDirection.TARGET;
+import static info.rmapproject.indexing.TestUtils.getRmapObjects;
```

Comment:
Is it possible to only make the necessary changes to import statements?  When looking at an IT and a bunch of changes to imports, I just worry about something being added or removed.  It's hard, when looking this diff, if that is the case.  So typically, if imports need to be changed, I wouldn't rely on the auto-formatting of the IDE (i.e. using "optimize imports"), but rather make only the necessary changes.